app/api/resources/mentorship_relation.py

```diff
@@ -3,6 +3,8 @@
 from flask_jwt_extended import jwt_required, get_jwt_identity
```

Comment:
Can you create a doc listing the rates for each endpoint or display the rate limit on the swagger UI?

Reply:
I will create the doc, but I will need help in deciding the rate limit for each endpoint

Reply:
You can ask it on zulip itself